Lib/test/test_textwrap.py

```diff
@@ -1076,5 +1077,21 @@ def test_first_word_too_long_but_placeholder_fits(self):
         self.check_shorten("Helloo", 5, "[...]")
 
 
+class WideCharacterTestCase(BaseTestCase):
+    def test_wide_character(self):
+        def text_len(text):
+            n = 0
+            for c in text:
+                if unicodedata.east_asian_width(c) in {'F', 'W'}:
+                    n += 2
+                else:
+                    n += 1
+            return n
+
+        text = "123 🔧"
+        expected = ["123", "🔧"]
+        self.check_wrap(text, 5, expected, text_len=text_len)
```

Comment:
I feel that more tests are needed; text_len is used at multiple places in the code.

Reply:
Thanks for adding tests!  Could some of them use different texts?

Reply:
I would like to avoid adding example for languages that I do not speak myself (e.g. Chinese). Are there ways to reach out to the wider community to provide meaningful examples?